### ISSUE TITLE
3.03 Release Uptade

### ISSUE THREAD

**Issue by Alakabon** [GregRos/IEMod.pw#93]
Here's the new DLL, with the console commands fix, updated for 3.03:

[IEMod.pw.dll.zip](https://github.com/IEMod/IEMod.pw/files/367414/IEMod.pw.dll.zip)

This should fix the combat crashes, but it breaks custom commands, meaning you can only use normal commands in the vanilla game (without disabling achievements or  having to type iroll20s ) . Later down the line I might end up restoring them but for the moment it's not a priority.

I also remade the spells per encounter module, it should work just fine.

If it crashes, please let me know here, I don't look at the nexus page often.

Known issues:
- Drop-down menu options are ''Resolving'' the first time the IEMOD options tab is opened, but fine the 2nd.... not sure how to fix that
- Camera scroll in menus still present
- Sometimes the voices are delayed and you can only start hearing them halfway through

MAKE SURE that you delete the old "Assembly-CSharp.dll.pw.modified" file in your managed folder and manually remove-add the new DLL in patchwork, otherwise it might not recompile it before launching.

**Comment by jasonlejardo** [comment]
Thanks!
Using this one on game 3.03 for 2-3 days and works very good!

**Comment by ALIENQuake** [comment]
Many thanks!

**Comment by ALIENQuake** [comment]
@Alakabon Many thanks for this release!

**Comment by Alakabon** [comment]
There is a newer version of it on the nexus. I've been given acces to the page and have been updating it relatively steadily and fixing bugs/adding a bit of features here and there.

**Comment by ALIENQuake** [comment]
@Alakabon Yes, I've use nexusmods release. Maybe leave github releases only for beta version?

**Comment by Alakabon** [comment]
That would make sense, but since it's fixing bugs it feels like I'm letting people down until the whole release... Not many people will see the github. I don't have permissions here either, the pull request for the 3.03 patch update hasn't been approved yet.
